router/config/literals.js

```diff
@@ -200,11 +200,11 @@ message.CHOOSE_STATE_BOARD =
     }
 }
 
-message.TN_BOARD =
+message.NISHTHA_BOARD =
 {
     "type": "button",
     "data": {
-        "text": " Please visit <a target='_blank' href='https://diksha.gov.in/tn/explore/?utm_source=Tara'>DIKSHA TAMILNADU</a><br>Select the Medium and Class to view relevant subject textbook on the board website.<br>",
+        "text": " Please visit <a target='_blank' href='https://diksha.gov.in/explore/1?key=%key%&selectedTab=all&board=%board%'>DIKSHA %board%</a><br>Select the Medium and Class to view relevant subject textbook on the board website.<br>",
```

Comment:
Add utm_source=Tara' so that we can identify user came from chatbot

Reply:
If I add `&utm_source=Tara` route won't work
https://diksha.gov.in/explore/1?key=Nishtha%202020&selectedTab=all&board=state%20(andhra%20pradesh)&utm_source=Tara